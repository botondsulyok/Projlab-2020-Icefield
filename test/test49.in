field stable 0
entity esk Eskimo stable
item tent1 Tent esk
tent esk
item tent2 Tent esk
stats stable
stats esk
tent esk
stats stable
stats esk