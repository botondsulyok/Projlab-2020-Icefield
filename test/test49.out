id: stable
type: iceunit
building: tent1
snow: 0
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: null

id: esk
type: eskimo
iceUnit: stable
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: tent2
components: null

id: stable
type: iceunit
building: tent1
snow: 0
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: null

id: esk
type: eskimo
iceUnit: stable
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: tent2
components: null

